brainrender/scene.py

```diff
@@ -473,12 +478,35 @@ def add_root(self, render=True, **kwargs):
 
 		return self.root
 	
-	def add_brain_regions(self, *args, **kwargs):
+	def add_brain_regions(self, brain_regions,  *args, c=None,
+						  use_original_color=True,
+						  alpha=0.4, **kwargs):
 		"""
 			Adds brain regions meshes to scene.
 			Check the atlas' method to know how it works
 		"""
-		return self.atlas.add_brain_regions(self, *args, **kwargs)
+		actors = []
```

Comment:
I'll need to fix these methods first. Will do it tomorrow, I'll probably do something similar to this, with some differences. Not sure how it's going to work for methods like 'add_neurons'